.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,23 @@
+## For issue
+
 ## Please verify the following:
 
 - [ ] `yarn test` **jest** tests pass with new tests, if relevant
 - [ ] `yarn lint` **eslint** checks pass with new code, if relevant
 - [ ] `yarn format:check` **prettier** checks pass with new code, if relevant
 - [ ] `README.md` (or relevant documentation) has been updated with your changes
+- [ ] If this affects functionality there aren't tests for, I manually tested it, including by generating a new app locally if needed ([see docs](https://docs.infinite.red/ignite-cli/contributing/Contributing-To-Ignite/#testing-changes-from-your-local-copy-of-ignite)).
 
 ## Describe your PR
+
+<!-- If this PR addresses an issue, link to it in description: "Addresses #333" -->
+
+[Add your PR description]
+
+## Screenshots (if applicable)
+
+If you’re submitting code changes related to a visible feature, please include before-and-after screenshots or videos.
```

Comment:
Could we include an example of HTML markup for resizing images that people could reference.

```html
<!-- If GH's auto attachment previews too large, trying resizing it with: <img src="filename-from-github-upload" width="300" /> -->
```

Reply:
In my experience and within a markdown table, Github resizes the image to fit the space. And works also when having multiple columns.

Reply:
But also, some people can delete the table and add the image directly, which in that case could be a large one.

Reply:
for sure, maybe just including this as a comment might be useful to people who want quick access to how to do it. otherwise it can be ignored.